spec/02-integration/17-admin_gui/01-admin-gui-path_spec.lua

```diff
@@ -0,0 +1,154 @@
+local lfs = require "lfs"
+local pl_path = require "pl.path"
+local helpers = require "spec.helpers"
+local test_prefix = helpers.test_conf.prefix
+
+local _
+
+describe("Admin GUI - admin_gui_path", function()
+  local client
+
+  after_each(function()
+    helpers.stop_kong()
+    if client then
+      client:close()
+    end
+  end)
+
+  lazy_teardown(function()
+    helpers.stop_kong()
+  end)
+
+  it("should serve Admin GUI correctly when admin_gui_path is unset", function()
+    assert(helpers.start_kong({
+      database = "off",
+      admin_gui_listen = "127.0.0.1:9002",
+    }))
+
+    local err, gui_dir_path, gui_index_file_path, gui_config_dir_path, gui_config_file_path
+    gui_dir_path = pl_path.join(test_prefix, "gui")
+    os.execute("rm -rf " .. gui_dir_path)
+    _, err = lfs.mkdir(gui_dir_path)
+    assert.is_nil(err)
+
+    gui_config_dir_path = pl_path.join(test_prefix, "gui_config")
+    os.execute("rm -rf " .. gui_config_dir_path)
+    _, err = lfs.mkdir(gui_config_dir_path)
+    assert.is_nil(err)
+
+    gui_index_file_path = pl_path.join(gui_dir_path, "index.html")
+    gui_config_file_path = pl_path.join(gui_config_dir_path, "kconfig.js")
```

Comment:
I think you should remove these codes here. Because there are no `kconfig.js` to be created.